### PR TITLE
Add additional DB unit tests

### PR DESCRIPTION
**Pull request by chetan-rns**
#### Description:
- Add additional unit tests for DB Queries

#### Link to JIRA Story (if applicable):

https://issues.redhat.com/browse/GITOPSRVCE-693

### CODE REVIEW

File: backend-shared/db/app_project_repository_test.go [hunk 1]
```diff
@@ -199,4 +200,117 @@ var _ = Describe("AppProjectRepository Test", func() {
...
+			testURL := "https://private-repo:6443"
+			appProjectRepository.RepoURL = testURL
+			appProjectRepository.AppprojectRepositoryID = ""
+
+			err := dbq.UpdateAppProjectRepository(ctx, appProjectRepository)
+			Expect(err).To(HaveOccurred())
+			Expect(err.Error()).To(Equal("appproject_repository_id field should not be empty string, in UpdateAppProjectRepository"))
+		})
+
+		It("should return an error if the cluster user ID ID field is not set", func() {
```
Comment (jopit) [inline backend-shared/db/app_project_repository_test.go:276]: `ID` is duplicated.

**Review by jopit — CHANGES_REQUESTED** [review]
Looks good, just one very minor change.

**Review by jopit — APPROVED** [review]
lgtm

**Review by jgwest — APPROVED** [review]
LGTM too, thanks @chetan-rns, and thanks @jopit for reviewing!

**Pull request merged**